config/software/delivery-cli.rb

```diff
@@ -69,8 +69,9 @@
       dlls << "libgcc_s_seh-1"
     end
     dlls.each do |dll|
-      arch_suffix = windows_arch_i386? ? "32" : "64"
-      windows_path = "C:/msys2/mingw#{arch_suffix}/bin/#{dll}.dll"
+      mingw = ENV["MSYSTEM"].downcase
```

Comment:
Will this be set in build nodes using legacy `omnbius-toolchain` and `load-omnbius-toolchain.bat`?